transform_stats_to_metrics.go

```diff
@@ -249,26 +261,26 @@ func bucketsToProtoBuckets(countPerBucket []int64) []*metricspb.DistributionValu
 	return distBuckets
 }
 
-func labelValuesFromTags(tags []tag.Tag) []*metricspb.LabelValue {
-	if len(tags) == 0 {
-		return nil
+func labelValuesFromTags(tags []tag.Tag, keysToColumns map[string]int) ([]*metricspb.LabelValue, error) {
+	// It is imperative that we set the "HasValue" attribute,
+	// in order to distinguish missing a label from the empty string.
+	// https://godoc.org/github.com/census-instrumentation/opencensus-proto/gen-go/metrics/v1#LabelValue.HasValue
+
```

Comment:
Please remove this line. That way we can anchor the comment right next to where it is used.